doc/usermanual/darkroom/modules/tone/filmicrgb.xml

```diff
@@ -0,0 +1,619 @@
+            B³)/(R² + G² + B²). It is usally a good compromise between the max RGB and the
+            luminance Y values.
+          </para></listitem>
+
+        </itemizedlist>
+      </para>
+      <para>
+        There is no "right" choice for the norm, depending on the picture to which it applies -
+        you should experiment and decide for yourself on case by case basis.
+      </para>
+    </sect5>
+
+    <sect5>
+<!-- IHU I haven't touched that section (anything in the display tab) because
+      I don't have the knowledge/competence to see if this was modified or not compared to the
+      previous version.-->
+      <title>Target black luminance (<guilabel>Display</guilabel> tab)</title>
+      <para>
+        The destination parameters set the target luminance values used to remap the tones
+        through filmic rgb. The default parameters will work 99% of the times, the remaining 1%
+        being when you output in linear RGB space (REC709, REC2020) for media handling
+        log-encoded data. These settings are then to be used with caution because darktable does
+        not allow separate pipelines for display preview and for file output.
+      </para>
+      <para>
+        The target black luminance parameter allows to set the ground-level black of the target
+        medium. Set it greater than 0% if you want raised, faded blacks to achieve a retro look.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Target middle-grey (<guilabel>Display</guilabel> tab)</title>
+      <para>
+        This is the middle-grey of the output medium, that is used as a target for the filmic
+        rgb S curve central node. On gamma corrected media, the actual grey is computed with the
+        gamma correction (middle-grey^(1/gamma)), so a middle-grey parameter of 18% with a gamma
+        of 2.2 gives an actual middle-grey target of 45.87%.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Target white luminance (<guilabel>Display</guilabel> tab)</title>
+      <para>
+        The target white luminance parameter allows to set the ceiling level white of the target
+        medium. Set it lower than 100% if you want dampened, muted whites to achieve a retro
+        look.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>Target power factor function</title>
+      <para>
+        The power of the output transfer function, often improperly called the gamma (only
+        screen have a gamma), is the parameter used to raise or compress the mid-tones to
+        account for the display non-linearities or to avoid quantization artifacts when encoding
+        in 8 bits file formats. This is a common operation when applying ICC color profiles
+        (except for linear RGB spaces, like REC 709 or REC 2020, which have a linear
+        <quote>gamma</quote> of 1.0). However, at the output of filmic rgb, the signal is
+        logarithmically encoded, which is not something ICC color profiles know to handle. As a
```

Comment:
logarithmically encoded, which is not something ICC color profiles know how to handle. As a